nilearn/datasets/atlas.py

```diff
@@ -782,3 +782,73 @@ def fetch_coords_dosenbach_2010(ordered_regions=True):
                   networks=out_csv['network'], description=fdescr)
 
     return Bunch(**params)
+
+
+def fetch_atlas_allen_2011(data_dir=None, url=None, resume=True, verbose=1):
+    """Download and return file names for the resting-state networks atlas
+    published by Allen et al. in 2011. The provided images are in MNI152 space.
+
+    Parameters
+    ----------
+    data_dir: str, optional
+        directory where data should be downloaded and unpacked.
+    url: str, optional
+        url of file to download.
+    resume: bool
+        whether to resumed download of a partly-downloaded file.
+    verbose: int
+        verbosity level (0 means no message).
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        dictionary-like object, keys are:
+        - "all_unthresh_tmaps": T-maps of all 75 unthresholded components.
+        - "rsn_unthresh_tmaps": T-maps of 28 RSNs included in E. Allen et al.
+        - "all_unthresh_rsn_labels": Labels for the 28 RSNs.
+        - "aggregate_ic_comps": Aggregate ICA Components.
```

Comment:
@alexsavio : sorry for the long turn around. I am doing too many things, and as a result I become a bottleneck.

It would be good if we could try to be as close as possible to the structure of the other atlases that we are downloading. In particular, it would be good if the maps that are most likely to be used would be in a key called "maps", as in the other atlases, and the labels in a list given the corresponding labels in the order of the maps. You would have to give names to maps, rather than networks, but that should be easy, for instance "Default-Mode_50", .... You would need to add a list of networks labels too. The names of the other maps could then use a similar terminology, such as "unthresh_maps"", and "networks_maps".

The msdl atlas, for instance, exposes this structure. In the beginning, we weren't very good at imposing this structure, but while teaching we saw that it led to unneeded conceptual difficulties for the users.